src/Chassis.Fairing.Mvc/DefaultRouting.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Web.Mvc;
+using System.Web.Routing;
+
+namespace Chassis.Fairing.Mvc
+{
+    public class DefaultRouting : IMvcStartupStep
+    {
+        public void Boot()
+        {
+            AreaRegistration.RegisterAllAreas();
+            var routes = RouteTable.Routes;
+
+            routes.IgnoreRoute("{resource}.axd/{*pathInfo}");
+
+            routes.MapMvcAttributeRoutes();

```

Comment:
i like where you are going with that, I'll have to noodle on that so its obvious and such.